ui/apps/platform/src/services/CloudSourceService.ts

```diff
@@ -0,0 +1,39 @@
+import axios from './instance';
+import { Empty } from './types';
+
+export type CloudSourceType = 'TYPE_UNSPECIFIED' | 'TYPE_PALADIN_CLOUD' | 'TYPE_OCM';
+
+export type CloudSourceCredentials = {
+    secret: string;
+};
+
+export type PaladinCloudConfig = {
+    endpoint: string;
+};
+
+export type CloudSourceIntegration = {
+    id: string;
+    name: string;
+    type: CloudSourceType;
+    credentials: CloudSourceCredentials;
+    skipTestIntegration: boolean;
+    paladinCloud?: PaladinCloudConfig;
```

Comment:
Aha, after MVP and before y’all add OCM integration, you can replace optional property with something like:

```ts
export type CloudSourceIntegration = PaladinCloudIntegration | OCMIntegration;

export type CloudSourceIntegrationBase = {
    id: string;
    name: string;
    type: CloudSourceType;
    credentials: CloudSourceCredentials;
    skipTestIntegration: boolean;
};

export type PaladinCloudIntegration = {
    type: 'TYPE_PALADIN_CLOUD',
    paladinCloud: PaladinCloudConfig;
} & CloudSourceIntegrationBase;

export type OCMIntegration = {
    type: 'TYPE_OCM',
    ocm: OCMConfig;
} & CloudSourceIntegrationBase;
```